### DISCUSSION TITLE
Rename the app from `AAIdrive` to `AAiDrive`

### DISCUSSION THREAD

**Discussion by SteWers** [BimmerGestalt/AAIdrive#845] [Ideas]
I suggest to rename the app from `AAIdrive` to `AAiDrive`.
In my eyes it looks better and BMW also writes `iDrive` with a lower `i` and a upper `D` on their web pages:

![image](https://github.com/user-attachments/assets/9bb0880d-abc7-41b0-89c0-21c4dea98509)

What do you think?

**Comment by harryberlin** [comment]
maybe a rights or licence issue.

**Comment by SteWers** [comment]
I don't think so. Legally there is no difference between upper or lower case letters. 
https://www.ratgeberrecht.eu/markenrecht-aktuell/gestaltung-einer-wortmarke.html

**Comment by hufman** [comment]
I like the current name, and the mixed case doesn't look as nice to me.
